jenkins-agents/jenkins-agent-cosign/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:4.8
```

Comment:
@ckavili  - I think the other agents are based on 4.7 so we might need to update those separately ....